editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -45,29 +46,35 @@ const send = (ops: Array<any>) => {
 export interface EditorUIStore {
   flowLayout: FlowLayout;
   showSidebar: boolean;
-  togglePreview: () => void;
+  toggleSidebar: () => void;
   isTestEnvBannerVisible: boolean;
   hideTestEnvBanner: () => void;
 }
 
 export const editorUIStore: StateCreator<
   SharedStore & EditorUIStore,
   [],
-  [],
+  [["zustand/persist", unknown]],
   EditorUIStore
-> = (set, get) => ({
-  flowLayout: FlowLayout.TOP_DOWN,
+> = persist(
+  (set, get) => ({
+    flowLayout: FlowLayout.TOP_DOWN,
 
-  showSidebar: true,
+    showSidebar: true,
 
-  togglePreview: () => {
-    set({ showSidebar: !get().showSidebar });
-  },
+    toggleSidebar: () => {
+      set({ showSidebar: !get().showSidebar });
+    },
 
-  isTestEnvBannerVisible: !window.location.href.includes(".uk"),
+    isTestEnvBannerVisible: !window.location.href.includes(".uk"),
 
-  hideTestEnvBanner: () => set({ isTestEnvBannerVisible: false }),
-});
+    hideTestEnvBanner: () => set({ isTestEnvBannerVisible: false }),
+  }),
+  {
+    name: "editorUIStore",
```

Comment:
Is Local Storage the default here or have I missed where you specified this?

Reply:
That's correct, local storage is just default so we don't need to specify anything here.